website/docs/manager_ftrack_actions.md

```diff
@@ -124,41 +110,23 @@ This action gives ability to *stop running jobs*. When action is triggered, an i
 With this action it's possible to delete up to 15 entities at once from active project in pipeline database. Entered names must match exactly the names stored in database. These entities also must not have children entities *(Sequence must not have Shots but Task is not entity)*.
 
 ---
-<div class="ImagePanel" data-image-align='right'>
-<div class="ImagePanel.content">
-
-## Thumbnail
-</div>
-<div class="ImagePanel.image">
-
-![thumbnail-icon](assets/ftrack/ftrack-thumbnail-icon.png)
-</div>
-</div>
-
-A group of actions for thumbnail management.
-
-### Thumbnail to Parent
-Propagates the thumbnail of the selected entity to its parent.
-
-### Thumbnail to Children
-Propagates the thumbnail of the selected entity to its first direct children entities.
-
----
-<div class="ImagePanel" data-image-align='right'>
-<div class="ImagePanel.content">
+<div class="row markdown">
+<div class="col col--10 markdown">
 
 ## Prepare Project
+
 </div>
-<div class="ImagePanel.image">
+<div class="col col--2 markdown">
 
 ![prepare_project-icon](assets/ftrack/ftrack-prepare_project-icon.png)
+
 </div>
 </div>
 
 * Entity types: Project
 * User roles: Pypeclub, Administrator, Project manager
 
-Allows project managers and coordinator to *set basic project attributes* needed for OpenPype to operate, *Create project folders* if you want and especially *prepare Project specific [anatomy](admin_config#anatomy) and [presets](admin_config#presets)* files for you.
+Allows project managers and coordinator to *set basic project attributes* needed for OpenPype to operate, *Create project folders* if you want and especially prepare project specific [anatomy](admin_settings_project_anatomy) or [settings](admin_settings_project).
```

Comment:
Let's remove the mention of anatomy, that will need to be done in settings. plus it's not actually being copied from 3.0